pure_pursuit_wrapper/src/pure_pursuit_wrapper.cpp

```diff
@@ -95,4 +100,34 @@ std::vector<double> PurePursuitWrapper::apply_response_lag(const std::vector<dou
   output = trajectory_utils::shift_by_lookahead(speeds, (unsigned int) lookahead_count);
   return output;
 }
+
+std::vector<cav_msgs::TrajectoryPlanPoint> PurePursuitWrapper::remove_repeated_timestamps(const std::vector<cav_msgs::TrajectoryPlanPoint>& traj_points){
```

Comment:
This assumes that all repeated timestamps are sequential, e.g. it catches [1, 2, 3, 4, 4, 5, 6, 7] but it does not catch [1, 2, 3, 2, 3, 4], just want to double check that the latter is not an expected case or worth handling. Might be worth documenting explicitly, since the function name would lead me to potentially believe otherwise.

Reply:
I dont think we ever saw timestamps that go back, so I have documented explicitly.